tests/rptest/tests/cluster_config_test.py

```diff
@@ -1396,22 +1396,32 @@ def test_status_read_after_write_consistency(self):
     aliased_name: str  # this is the legacy name, retained as an alias for backward compat
     redpanda_version: RedpandaVersionLine  # this is the first version to use primary_name
     test_values: list[Any, Any, Any]  # values for this property to run the tests
+    expect_restart: bool # setting property will ask for a restart
 """
-PropertyAliasData = namedtuple(
-    "PropertyAliasData",
-    ["primary_name", "aliased_name", "redpanda_version", "test_values"])
+PropertyAliasData = namedtuple("PropertyAliasData", [
+    "primary_name", "aliased_name", "redpanda_version", "test_values",
+    "expect_restart"
+])
 
 cloud_storage_graceful_transfer_timeout = PropertyAliasData(
     primary_name="cloud_storage_graceful_transfer_timeout_ms",
     aliased_name="cloud_storage_graceful_transfer_timeout",
     redpanda_version=(23, 2),
-    test_values=(1234, 1235, 1236))
+    test_values=(1234, 1235, 1236),
+    expect_restart=False)
 log_retention_ms = PropertyAliasData(primary_name="log_retention_ms",
                                      aliased_name="delete_retention_ms",
                                      redpanda_version=(23, 3),
-                                     test_values=(1000000, 300000, 500000))
+                                     test_values=(1000000, 300000, 500000),
+                                     expect_restart=False)
 # NOTE due to https://github.com/redpanda-data/redpanda/issues/13432 ,
 # test_values can't be -1 (a valid value nonetheless to signal infinite value)
```

Comment:
Is this the right spot for this comment?

Reply:
Right it makes more sense before log_retention_ms. Will do in a followup

Reply:
SGTM